iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -259,8 +260,9 @@ public synchronized void updateAllExistingReportedProperties()
             }
         }
 
-        public synchronized void updateExistingReportedProperty(int index)
+        public synchronized Set<Property> updateExistingReportedProperty(int index)
```

Comment:
this method is no longer doing what its name implies does it? it's creating and returning a new set

Reply:
It still does what it says. It only returns the property that was updated, but that returned property did already exist

Reply:
By the way, this and other methods always return Set<Property> even when only one property is in the set. The reason for this is so that the test method can use this returned value directly when calling "sendReportedProperties(Set<Property>)" instead of building the set after calling this method

Reply:
I see, so it updates the existing and also returns a new set as a return value. fair.